AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Sample_Editor_Tests.py

```diff
@@ -0,0 +1,143 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Test Case: Creating tests for basic editor feature like creating a level, adding new entities, modifying the entities etc.
+"""
+
+
+class Tests:
```

Comment:
There are some tests here that are never actually called using Report.result. Additionally, I'd put them in the order they're found in the script just for usability/readability.

Reply:
fixed